src/main/resources/assets/iris/lang/zh_tw.json

```diff
@@ -44,11 +44,11 @@
   "options.iris.shadowDistance": "最大陰影距離",
   "options.iris.shadowDistance.enabled": "能讓您變更陰影的最大距離。超出此距離的地形和實體將不會有陰影。調低陰影距離能顯著提升效能。",
   "options.iris.shadowDistance.disabled": "您無法變更目前光影包所設定的陰影顯示距離。",
-  "options.iris.shadowDistance.sodium_tooltip": "「陰影顯示距離」選項能控制在陰影傳遞（shadow pass）時，可能呈現的地形有多遠。 較近的距離將呈現較少的地形，但畫面播放速率（frame rates）也得以增加。 此選項在套用「有特定陰影顯示距離的光影包」時無法變更。 實際的陰影顯示距離會受「顯示距離」選項限制。",
+  "options.iris.shadowDistance.sodium_tooltip": "「陰影顯示距離」選項能控制在陰影傳遞（ shadow pass）時，可能呈現的地形有多遠。 較近的距離將呈現較少的地形，但畫面播放速率（frame rates）也得以增加。 此選項在套用「有特定陰影顯示距離的光影包」時無法變更。 實際的陰影顯示距離會受「顯示距離」選項限 制。",
```

Comment:
what's the need for the extra space in the parentheses?

Reply:
For readability.
To show the difference, screenshots attached
![without](https://user-images.githubusercontent.com/41604849/149852309-dd82ef47-8880-4072-bfeb-8490f080621f.png)
* Without the extra space.  The "w" of the word shadow goes to the newline. The Chinese period does that too.

Reply:
Thanks for the explanation, makes sense.